core/components/molecules/dialog/dialog.js

```diff
@@ -34,75 +35,165 @@ const createButtonForAction = (action, index) => {
   )
 }
 
-const Dialog = props => (
-  <Overlay {...props}>
-    <DialogElement width={props.width} {...Automation('dialog')}>
-      <DialogTitleBar {...Automation('dialog.title')}>
-        <span>{props.title}</span>
-        <Link onClick={props.onClose}>
-          <Icon name="close" size={16} />
-        </Link>
-      </DialogTitleBar>
-      <DialogBody {...Automation('dialog.body')}>{props.children}</DialogBody>
-      <DialogFooter {...Automation('dialog.footer')}>
-        <ButtonGroup>{props.actions.map(createButtonForAction)}</ButtonGroup>
-      </DialogFooter>
-    </DialogElement>
-  </Overlay>
-)
-
-const DialogElement = styled.div`
+const focusOnFormInput = ({ current }) => {
```

Comment:
This is pretty form + dialog specific code. Can this be done with an `autoFocus` prop by the user instead?

Reply:
This is the idea, for us to force this to play well with the other dialog-specific accessibility enhancements.

Reply:
Sorry, I don't understand what you mean. Can you add a comment in the code explaining it?

Reply:
Hi :wave: 

Saw this was resolved, but couldn't find the comment